BUILDING

```diff
@@ -0,0 +1,60 @@
+Installing
+----------
+
+To build and install tarsnap, run:
+    ./configure
+    make
+    make install
+
+Tarsnap should build and run on any IEEE Std 1003.1 (POSIX) compliant
```

Comment:
s/any/most/

In particular, last time I checked Tarsnap didn't build on GNU Hurd because they don't define PATH_MAX.